### ISSUE TITLE
Add more tests with non English documents

### ISSUE THREAD

**Issue by rth** [FreeDiscovery/FreeDiscovery#60]
It would be helpful to add a document collection with non English document (possibly even with a non latin alphabet), and validate how FreeDiscovery behaves with it. 

The next (and more complicated) step would be to handle Chinese / Japanese documents, which would require a dedicated work tokenizer (as words would not be separated by spaces anymore).

**Comment by dmytrobondarchuk** [comment]
Hello. I created a collection of documents that contain Cyrillic to validate how FreeDiscovery behaves with it. This set is based on the `TREC Legal Collection 2009` and actually contains documents from the `treclegal09_2k_subset` translated in Russian. The results of the work FreeDiscovery with this set I attached. Please look on them.
[log_REST_categorization.txt](https://github.com/FreeDiscovery/FreeDiscovery/files/848887/log_REST_categorization.txt)
[log_REST_clustering.txt](https://github.com/FreeDiscovery/FreeDiscovery/files/848888/log_REST_clustering.txt)
[log_REST_data_ingestion.txt](https://github.com/FreeDiscovery/FreeDiscovery/files/848891/log_REST_data_ingestion.txt)
[log_REST_duplicate_detection.txt](https://github.com/FreeDiscovery/FreeDiscovery/files/848889/log_REST_duplicate_detection.txt)
[log_REST_semantic_search.txt](https://github.com/FreeDiscovery/FreeDiscovery/files/848890/log_REST_semantic_search.txt)

**Comment by rth** [comment]
Thanks for looking into it! 

Are you sure that your modified dataset was loaded? it looks like 
```
GET http://localhost:5001/api/v0/example-dataset/treclegal09_2k_subset
```
in the examples still loads the original `treclegal09_2k_subset` (cluster labels are also in English while they shouldn't be).   It might be simpler to just modify the path in `datasets.load_dataset` function (in some temporary branch), if you want to run examples unchanged on a different dataset? 

Alternatively we could add just a 3-5 modified documents, for instance, under `data/ds_002/raw` then add another unit test under `freediscovery/tests/test_text.py` (similar to `test_feature_extraction_tokenization`) to test that feature extraction works on these documents. If feature extraction doesn't raise errors, the rest should be fine (as we are manipulating numeric arrays).  This would also be a useful test to have if we ever add other feature extraction methods..

**Comment by dmytrobondarchuk** [comment]
Thanks! You're right.  I just extracted the modified dataset (https://yadi.sk/d/zxgPepaj3G7Xzp) to `cache_dir` and then run examples unchanged. It's a good idea to add just a 3-5 modified documents and another unit test under `freediscovery/tests/test_text.py`. I'll impliment this.

**Comment by rth** [comment]
Thank you ! Sounds great..

**Comment by dmytrobondarchuk** [comment]
Hi. I added 6 modified documents under `data/ds_002/raw` and  another unit test under `freediscovery/tests/test_text.py` to test that feature extraction works on these documents. The feature extraction doesn't raise errors.

**Comment by rth** [comment]
Fixed in #120 .
